BUILD

```diff
@@ -8,7 +8,7 @@ filegroup(
 genrule(
     name = "gen-std-jsonnet-h",
     srcs = ["stdlib/std.jsonnet"],
-    outs = ["core/std.jsonnet.h"],
+    outs = ["std.jsonnet.h"],
```

Comment:
I was getting an Bazel build error about being unable to find the generated header file:

```
❯❯❯ bazel build //:jsonnet-common
INFO: Found 1 target...
ERROR: /Volumes/Ocean/Projects/google/jsonnet/BUILD:17:1: undeclared inclusion(s) in rule '//:jsonnet-common':
this rule is missing dependency declarations for the following files included by 'core/desugarer.cpp':
  'bazel-out/local_darwin-fastbuild/genfiles/std.jsonnet.h'.
Target //:jsonnet-common failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 1.457s, Critical Path: 1.29s
```

This might be a bug in the C/C++ rules related to using the `includes` attribute while looking for a header file in the genfiles tree. Changing the genrule to output the file in the current package rather than `core/` worked.